books/free-programming-books-he.md

```diff
@@ -0,0 +1,36 @@
+### Index
+
+* [C#](#cs)
+* [Assembly](#assembly)
+* [Python](#python)
+* [ללא תלות בשפה](#ללא-תלות-בשפה)
+  * [מערכות הפעלה](#מערכות-הפעלה)
+  * [רשתות](#רשתות)
```

Comment:
Could you adapt this:

```markdown
* [ללא תלות בשפה](#ללא-תלות-בשפה)
  * [מערכות הפעלה](#מערכות-הפעלה)
  * [רשתות](#רשתות)
```

to follow more or less the style of the other listings? I'm not be able to RTL write with numbers.

- 0 - Metalists
  * book
- 1 - Agnostic to language
  * book | section
- Assembly

E.g: https://github.com/EbookFoundation/free-programming-books/blob/master/books/free-programming-books-es.md

I mean, to move it as the first listing section.

If it's not posible, doesn't matter

Reply:
Done by commit 9e1c741